celery/utils/functional.py

```diff
@@ -389,3 +389,7 @@ def seq_concat_seq(a, b):
     if not isinstance(b, prefer):
         b = prefer(b)
     return a + b
+
+
+def is_numeric_value(value):
```

Comment:
this function also need test coverage.

Reply:
Done! I added the test (with several test cases) in https://github.com/celery/celery/pull/7291/commits/0e8d9db9a1ecc9a3192b647b690fec7b482f778a